packages/opentelemetry-cloud-monitoring-exporter/package.json

```diff
@@ -39,11 +39,11 @@
   "devDependencies": {
     "@types/mocha": "7.0.2",
     "@types/nock": "11.1.0",
-    "@types/node": "12.12.47",
     "@types/sinon": "9.0.4",
     "codecov": "3.7.0",
-    "gts": "2.0.2",
+    "gts": "1.1.0",
```

Comment:
What's the reason to use this older version?

Reply:
When the CI runs for linting these are the modules installed:
+ lerna@3.22.1
 
+ gts@1.1.0
 
+ typescript@3.9.5
 
I thought it would make sense for the local linting to use the same version of gts as the CI.

Reply:
I see, looks like it was pinned in https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/36 which links to some instructions for upgrading. If you got it working on 2.x, I don't see any reason to not upgrade?

Reply:
My original thinking was that going forward it might make sense to have the local version of gts tied to the one in CI in case anything comes up locally which would not be considered a "real issue" by the CI linting. Currently there exist no discrepancies so I agree there is no reason not to keep it at 2.0.2, I reverted it.

Reply:
I meant upgrade it in the CI config as well. If there are no issues with the new version, we can remove that 1.1.0 dep from the CI lint config so local and CI do the same thing, right?